csr/csr.py

```diff
@@ -20,9 +20,6 @@ class Individual(BaseModel):
     ic_version: Optional[float]
     ic_given_date: Optional[date]
     ic_withdrawn_date: Optional[date]
-    ic_material: Optional[str]
```

Comment:
Please do not remove fields, as this will break saved queries and notifications. From the [pipeline documentation](https://docs.google.com/document/d/1WiZMTviHP555uwjKWn1TpkUJ3vy6fjvu2FaYToZVJjg/edit#heading=h.ymv7pea9omsw):
> NOTE! It is recommended to only add fields to existing entities or add new entities. Changing existing fields will cause inconsistencies and may break saved queries and subscriptions.

Reply:
Roger. Is this currently an issue or more so once we get into production?

Reply:
The latter, so for now it is possible.

Reply:
Just testing if the documentation is accurate :)

Reply:
I think the warning is accurate ;-). Death to those members!